app/models/micropost.rb

```diff
@@ -0,0 +1,3 @@
+class Micropost < ApplicationRecord
+  scope :posts, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent, ->{order(created_at: :desc)}
```
Đổi tên lại cho đúng ý nghĩa